src/uhttp.c

```diff
@@ -32,6 +32,10 @@ static const char* HTTP_HOST = "Host";
 static const char* HTTP_CONTENT_LEN = "content-length";
 static const char* HTTP_TRANSFER_ENCODING = "transfer-encoding";
 static const char* HTTP_CRLF_VALUE = "\r\n";
+static const char* HEADER_ENDING = "\x0A";
+static const char* AUTH_HEADER = "Authorization:";
+static const char* AUTH_HEADER_TWO = "\nAuthorization";
```

Comment:
```suggestion
static const char* AUTH_HEADER_TWO = "\nAuthorization:";
```
should this have the colon as well?

Reply:
Good point! Fixed.